api-report/local-driver.api.md

```diff
@@ -101,7 +101,7 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
     // (undocumented)
     readBlob(blobId: string): Promise<ArrayBufferLike>;
     // (undocumented)
-    get repositoryUrl(): string;
+    readonly repositoryUrl: string;
```

Comment:
I think this change is safe to make in main even though it changes the API "shape" a little. I don't think users will need to update code to account for this change - it should just work because the forms are equivalent.

Reply:
That said, I reverted this.